### PR TITLE
[WIP] UI cleanup - Round 1

### PR DESCRIPTION
**Pull request by rodrigoA**
- Revisted pretty much all dialogs and editors and organized their layouts when needed
- Main window widgets were put in thinner layouts. Now we have more space
- Created an UI file for EquipmentEditor
- Created some proxy filters. These are related to new layouts and functionalities.
- All dialogs now have an Ok or Save and Ok buttons.
- Updated some missing tooltips

Note 1:  Mash designer was not modified in this round.
Note 2: The name of my branch does not reflect the changes that were made. My bad ... :(

### CODE REVIEW

**Comment by rodrigoA** [comment]
Hope you guys enjoy these small changes.
If you have suggestions/critics please don't hesitate, tell me.

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -180,6 +182,7 @@ SET( brewtarget_UIS
     ${UIDIR}/mashStepEditor.ui
     ${UIDIR}/miscEditor.ui
     ${UIDIR}/fermentableEditor.ui
+    ${UIDIR}/equipmentEditor.ui
```
Comment (rocketman768) [inline src/CMakeLists.txt]: So, my effort to murder equipmentEditor.ui was in vain, and it is risen from the grave?

Comment (rodrigoA) [inline src/CMakeLists.txt]: Didn't notice it before. I can revert the changes.
Why have you killed it?
When I saw it, I just thought the opposite! haha
"The guys are moving from programatically created ui to .ui files."

**Comment by rodrigoA** [comment]
No longer a mockup.

I´m pretty sure I´m not inadvertently creating new entries on database when combobox index change, but if some database expert is excited about checking this out, plz go ahead. :)

sem titulo

**Comment by rocketman768** [comment]
Is this one ready after the conflicts are fixed?

**Comment by mikfire** [comment]
What @rocketman768 asked. I would not mind including this in the 2.4 release.

**Comment by pricelessbrewing** [comment]
@rodrigoA What's the status on this? Are you still working on resolving conflicts? I think everyone appreciates the UI changes you made and would like to see them included in the next release.

**Comment by rodrigoA** [comment]
Hey @pricelessbrewing, I'm not.
@rocketman768  please delete this pull request.
Sorry guys.

**Comment by rodrigoA** [comment]
Guys, need a little help here.
I was working in a Fedora VM box during that time (2016). It happened that I bought another computer and the VM was copied to a external drive. I completely forgot about it...
Today, looking for other stuff (non-related) I found the folder with my old virtual dev boxes.
The git repository is still there with some pending commits LoL.
So, is it possible to reconfigure the upstream using a new fork?

@pricelessbrewing If I succeed re-configuring ,then yes, I'll finish what I have started.

**Pull request closed without merging**